package.json

```diff
@@ -36,7 +36,7 @@
         "react-beautiful-dnd": "^13.1.1",
         "react-csv-downloader": "^3.1.0",
         "react-dom": "^18.2.0",
-        "react-flow-renderer": "^10.3.17",
+        "reactflow": "^11.11.4",
```

Comment:
11 or 12 ?

Reply:
this PR is not finished yet, i started it from https://github.com/gridsuite/gridstudy-app/pull/2300 to continue on the migration